scripts/update_online.py

```diff
@@ -0,0 +1,27 @@
+import socket
+import os
+def arg(p=1):
+  try:
+    import sys
+    return sys.argv[p]
+  except:
+    return None
+os.chdir("..")
+
+lists = ["porn.txt","abuse.txt","ads.txt","basic.txt","crypto.txt","drugs.txt","everything.txt","facebook.txt","fraud.txt","gambling.txt","malware.txt","phishing.txt","piracy.txt","ransomware.txt","redirect.txt","scam.txt","smart-tv.txt","tiktok.txt","torrent.txt","tracking.txt","whatsapp.txt","youtube.txt"]
+for list in lists:
+    file = open(list)
+    alt = open(list.split(".")[0] + "_lite.txt","w")
```

Comment:
What does the `,"w"` part mean?

Reply:
That sets the mode to write; without it, the computer will assume I wanted to read the file, and will not allow me to edit it